### ISSUE TITLE
Create new renamed clips (that have a marker applied) from a Timeline

### ISSUE THREAD

**Issue by samplue** [fcpxhacks/fcpxhacks#3296]
Create an event with "new" browser clips from an existing timeline. It should only contain clips that have a Chapter Marker on them. The clips should be renamed based on the Marker name, but keep all other metadata (Scene, Take, Notes, etc).

**Comment by latenitefilms** [comment]
"Marked Timeline Clips to New Event"?

This could potentially be added to Fast Collections, a CommandPost Toolbox, or even the CommandPost HUD.

**Comment by IAmVigneswaran** [comment]
There is a possibility to add this feature into MarkersExtractor. But it may take a while.

https://github.com/TheAcharya/MarkersExtractor/issues/64

**Comment by latenitefilms** [comment]
I think it would be easier to do as a new Swift app, than building into `MarkersExtractor`.

**Comment by IAmVigneswaran** [comment]
Yes, separate app would make sense. But how many apps does the user need? 😅
